pkgs/applications/editors/kakoune/revert-uncaught-exceptions.patch

```diff
@@ -0,0 +1,56 @@
+diff --git a/src/file.cc b/src/file.cc
```

Comment:
Would be good to have this be `git-format-patch` output with the “this reverts commit…” bit intact.

Reply:
Done.